draco/analysis/transform.py

```diff
@@ -1265,3 +1266,343 @@ def process_finish(self) -> Union[containers.SiderealStream, containers.RingMap]
         self.mixed_data = None
 
         return data
+
+
+class Downselect(task.SingleTask):
```

Comment:
You might consider merging some of the internals of this into `draco.core.containers.copy_datasets_filter` to generalise this (that one only works on a single axis).

Reply:
That's a good idea but I might push it off until post-revision_07